src/ChainedExceptionHandler.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Nord\Lumen\ChainedExceptionHandler;
 
-use Exception;
+use Throwable;
```

Comment:
This can't be changed without a major version bump (`Throwable` is not there in PHP 5.6)

Reply:
then bump major version :)